testing/features/components/callouts.feature

```diff
@@ -54,3 +54,13 @@ Feature: Callout components
       Given the language is Welsh
       Then a callout title and message are present
       And an "Adviser" label is present above the callout title
+
+  Rule: Nested Callouts
+    Scenario: Callouts are not altered when nested
+      Given there are Nested Callout components on the page
+      Then both callouts are rendered correctly
+
+    Scenario: Callouts must not break WCAG heading guidelines
```

Comment:
There is no code in the design system that changes the heading levels - it's all done in the public website here: https://github.com/citizensadvice/public-website/blob/develop/app/lib/formatters/heading_formatter.rb

The tests for this code should be in that repo (there are already spec tests).

This variant is just here for the VRTs and docs.

Reply:
I'll remove this scenario.